### PR TITLE
fix: Fix bank jump buttons in housing storage items

### PR DESCRIPTION
**Pull request by P0keDev**
Turns out I over-complicated the previous fix and made it break for non-bank storage menus--despite having fewer pages, all menus have the same quick jump buttons in vanilla Wynncraft, so the `getDefaultJumpDestinations` method wasn't necessary.

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java [hunk 1]
```diff
@@ -121,18 +121,14 @@ private void getCustomJumpDestinations() {
...
-            customJumpDestinations = getDefaultJumpDestinations();
+            switch (currentContainer) {
+                case BANK -> customJumpDestinations = QUICK_JUMP_DESTINATIONS;
+                case BLOCK_BANK -> customJumpDestinations = BLOCK_BANK_DESTINATIONS;
+                default -> customJumpDestinations = HOUSING_DEFAULT_DESTINATIONS;
+            }
         }
     }
 
-    private List<Integer> getDefaultJumpDestinations() {
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java]: I still think this refactor was nice, even if it is called from just one place.

**Comment by kristofbolyai** [comment]
I'll let @JamieCallan117 review this, I have not much idea what the correct behavior is.

**Comment by ShadowCat117** [comment]
Looks good, I don't have the issue with block bank anymore. Up to you if you want to keep the `getDefaultJumpDestinations` method, it does make it more obvious what it's doing so I'd say keep that

**Review by kristofbolyai — APPROVED** [review]

**Review by magicus — APPROVED** [review]

File: common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java]: I fixed it for ya :)

**Pull request merged**